doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md

```diff
@@ -154,18 +169,18 @@ spec:
   #     release: prometheus
   ```
 
-* `namespaceSelector` and `seletor` are used to select exporter's Kubernetes service. Because Ray uses a built-in exporter, the **ServiceMonitor** selects Ray's head service which exposes the metrics endpoint (i.e. port 8080 here).
+* **PodMonitor** in `namespaceSelector` and `selector` are used to select Kubernetes Pods.
   ```sh
-  kubectl get service -n default -l ray.io/node-type=head
-  # NAME                          TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                                         AGE
-  # raycluster-kuberay-head-svc   ClusterIP   10.96.201.142   <none>        6379/TCP,8265/TCP,8080/TCP,8000/TCP,10001/TCP   153m
+  kubectl get pod -n default -l ray.io/node-type=head
+  # NAME                            READY   STATUS    RESTARTS   AGE
+  # raycluster-kuberay-head-fj78s   0/1     Running   0          36s
   ```
 
-* `targetLabels`: We added `spec.targetLabels[0].ray.io/cluster` because we want to include the name of the RayCluster in the metrics that will be generated by this ServiceMonitor. The `ray.io/cluster` label is part of the Ray head node service and it will be transformed into a `ray_io_cluster` metric label. That is, any metric that will be imported, will also contain the following label `ray_io_cluster=<ray-cluster-name>`. This may seem optional but it becomes mandatory if you deploy multiple RayClusters.
+* `relabelings` replaces the label `__meta_kubernetes_pod_label_ray_io_cluster` with the new label name `ray_io_cluster` in the scraped metrics.
```

Comment:
It's a bit hard to understand. Can you clarify that this is meant to add the name of the RayCluster custom resource that this Pod belongs to, to every metric scraped by this PodMonitor?

It would be helpful if you can show a real metric with the label `ray_io_cluster`.